### ISSUE TITLE
Rocky 8.6 kickstart stalls with root password issue

### ISSUE THREAD

**Issue by anothercloudnative** [khuedoan/homelab#63]
### Describe the bug

Deploying with the Rocky 8.6 issues stalls complaining of Root account is disable.

- [x] I have read the document

### To reproduce

1. Run make tools, make configure, supply info for your environment
2. Run make
4. During kickstart installation stalls with "Root account is disabled"

### Expected behavior

Kickstart should continue normally, without user intervention.

### Screenshots

![image](https://user-images.githubusercontent.com/105527497/169719469-b45cc39a-f663-4b34-8f80-3aeb92d22e90.png)

### Additional context

You can move past this error by selecting option `8`, providing a password and then selecting `b` to begin installation..  Installation will complete but will also require user to press `enter` once installation is complete.  Supplying a root password in the kickstart template below, appears to allow the installation to complete as expected. 

`/homelab/blob/master/metal/roles/pxe_server/templates/kickstart.ks.j2` 

using the following line:

`rootpw --iscrypted <hashed password>`

where hashed password is created using  something like:

` python -c 'import crypt,getpass;pw=getpass.getpass();print(crypt.crypt(pw) if (pw==getpass.getpass("Confirm: ")) else exit())'`

**Comment by anothercloudnative** [comment]
Sorry, I missed the 8.

**Comment by khuedoan** [comment]
Confirm that the 8.6 upgrade caused this, it's strange that they included a breaking change in a minor version.
You can temporarily downgrade to 8.5 while waiting for the fix:

```yaml
iso_url: "https://download.rockylinux.org/pub/rocky/8.5/isos/x86_64/Rocky-8.5-x86_64-minimal.iso"
iso_checksum: "sha256:4eb2ae6b06876205f2209e4504110fe4115b37540c21ecfbbc0ebc11084cb779"
```

**Comment by ckstevenson** [comment]
@khuedoan I ran into this issue and changed the `iso_{url,checksum}` per the advice here, but it still had the same problem where the Kickstart requires user intervention. I confirmed that the ISO was indeed 8.5 by watching the boot sequence and even went as far as deleting containers, the iso in the data dir, etc. It appears you tested the 8.5 iso as well, so now I am wondering if you have any advice on what I can do to work around this?

**Comment by khuedoan** [comment]
@ckstevenson could you try again after deleting everything in `metal/roles/pxe server/files/data/os` (excluding `.gitignore`)? I believe this is a bug because the ISO file is not re-extracted when it changes.

**Comment by ckstevenson** [comment]
> @ckstevenson could you try again after deleting everything in `metal/roles/pxe server/files/data/os` (excluding `.gitignore`)? I believe this is a bug because the ISO file is not re-extracted when it changes.

Ah, of course! It is working now, thanks!

**Comment by khuedoan** [comment]
Reverted to `v8.5` for now: https://github.com/khuedoan/homelab/commit/0e1ff60b101a671f4c8357e5b3383eb9e958edda

**Comment by khuedoan** [comment]
Permanent fix for `v8.6`: https://github.com/khuedoan/homelab/commit/3b0b934df402868a67ed5fc9bdae7718f170ca35
